oracle/datadog_checks/oracle/oracle.py

```diff
@@ -78,29 +81,29 @@ def check(self, instance):
             self.use_oracle_client = False
             self.log.debug('Oracle instant client unavailable, falling back to JDBC: {}'.format(e))
 
-        with closing(self._get_connection(server, user, password, service, jdbc_driver, tags)) as con:
+        self.service_check_tags = ['server:%s' % server]
```

Comment:
We can't assign instance attributes on Agent 5 as all check instances use one class instance

Reply:
OK I'll change that.

Reply:
Removed!